react/src/pages/upload/UploadUtils.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+    DataEntryHeader,
+    DataEntryRow,
+    getDataEntryHeaders,
+    getProp,
+    snakeCaseToTitle,
+} from "../utils";
+
+// Convert a field string (snake_case) into a displayable title (Snake Case)
+export function formatFieldToTitle(field: string): string {
+    return snakeCaseToTitle(field) // convert to title case
+        .replace(/([iI][dD])/g, txt => txt.toUpperCase()); // capitalize any occurrance of "ID"
+}
+
+// Given a DataEntryRow field, return a new DataEntryHeader obj
+export function toColumn(
```

Comment:
`toColumn` is only used below without the title argument and `formatFieldToTitle` is only used here, so I'm assuming this is preparation for later?